simpleclient/src/main/java/io/prometheus/client/Histogram.java

```diff
@@ -207,6 +208,24 @@ public double time(Runnable timeable) {
       return elapsed;
     }
 
+    /**
+     * Executes callable code (i.e. a Java 8 Lambda) and observes a duration of how long it took to run.
```

Comment:
e.g.

Reply:
I see. Then on old method this should also be corrected, as I copied text from there.

Reply:
Yes, I must have missed that back when that was added.